lesson06/script.js

```diff
@@ -1,16 +1,16 @@
-const promise = new Promise(resolve => {
+const createList = new Promise(resolve => {
     const lists = [
         {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
         {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
     ];
-    
+
     setTimeout(() => resolve(lists),3000);
 });
 
-promise.then((value) => {
+createList.then((value) => {
     const fragment = document.createDocumentFragment();
     for(const list of value){
-        const li = document.createElement('li');
+     const li = document.createElement('li');
         const a = document.createElement('a');
         const img = document.createElement('img');
         a.textContent = list.text;
```

Comment:
I change variable name.
Because meaningful names are easy to understand.

Reply:
I'll post a link that might be helpful.
https://qiita.com/KeithYokoma/items/2193cf79ba76563e3db6

Reply:
Thank you for teaching me.
I'll try reading it!